scanner/services/matcher.go

```diff
@@ -98,8 +96,19 @@ func (s *matcherService) parseIndexReport(contents *v4.Contents) (*claircore.Ind
 	return ir, nil
 }
 
-func (s *matcherService) GetMetadata(_ context.Context, _ *types.Empty) (*v4.Metadata, error) {
-	return nil, status.Error(codes.Unimplemented, "method GetMetadata not implemented")
+func (s *matcherService) GetMetadata(ctx context.Context, _ *types.Empty) (*v4.Metadata, error) {
+	lastVulnUpdate, err := s.matcher.GetLastVulnerabilityUpdate(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting last vulnerability update time: %w", err)
+	}
+
+	timestamp, err := types.TimestampProto(lastVulnUpdate)
```

Comment:
Sanity: It might not be relevant if the first update is part of the readiness check, but anyway, while the first update runs, what does this conversion yield?